src/main/AuthenticationResponse.ts

```diff
@@ -90,21 +112,28 @@ export default class AuthenticationResponse {
    *
    * @param jwt ID token to be validated
    * @param verifyOpts
+   * @param linkedDomainValidationMode The policy regarding linkedDomainValidation
    */
-  static async verifyJWT(jwt: string, verifyOpts: VerifyAuthenticationResponseOpts): Promise<VerifiedAuthenticationResponseWithJWT> {
+  static async verifyJWT(
+    jwt: string,
+    verifyOpts: VerifyAuthenticationResponseOpts,
+    linkedDomainValidationMode?: LinkedDomainValidationMode
```

Comment:
Hmzz, why isn't this part of the actual Verify Options object?

Reply:
it is now